src/whylogs/app/logger.py

```diff
@@ -73,7 +70,7 @@ def __init__(self,
         self.dataset_name = dataset_name
         self.writers = writers
         self.verbose = verbose
-        self.cache = cache
+        self.cache = cache_size
```

Comment:
I would keep the same name, either direction.

Reply:
Thanks. I missed this